src/main/java/pro/sky/telegrambot/service/MessagesSender.java

```diff
@@ -0,0 +1,38 @@
+package pro.sky.telegrambot.service;
+
+import com.pengrad.telegrambot.TelegramBot;
+import com.pengrad.telegrambot.request.SendMessage;
+import com.pengrad.telegrambot.response.SendResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import pro.sky.telegrambot.listener.TelegramBotUpdatesListener;
+import pro.sky.telegrambot.model.NotificationTask;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Collection;
+
+@Service
+public class MessagesSender{
```

Comment:
отправляет нотификации/уведомления
```suggestion
public class NotificationSender {
```